spec/functions/deprecation_spec.rb

```diff
@@ -42,6 +42,7 @@
   end
 else
   describe 'deprecation' do
+    ENV['STDLIB_LOG_DEPRECATIONS'] = "true"
```

Comment:
This still needs to go in a `before` [hook](https://www.relishapp.com/rspec/rspec-core/docs/hooks/before-and-after-hooks), and there should be tests for each possible value(-set).